response_operations_ui/templates/collection-exercise.html

```diff
@@ -6,6 +6,18 @@
 <div role="main" id="main" class="page__main">
     <br/>
     <h1 class="jupiter" name="page-ce-title">{{survey.shortName}} {{ce.exerciseRef}}</h1>
+
+      {% if ce.state == "INIT" %}
```

Comment:
Rather than mapping the states in the template, I wonder if we should do something similar to [the PR for US011](https://github.com/ONSdigital/response-operations-ui/pull/35/files).